### PR TITLE
Fix settings for current VPC

### PR DESCRIPTION
**Pull request by miguelbernadi**

### CODE REVIEW

File: bin/lib [hunk 1]
```diff
@@ -166,12 +166,9 @@ function create() {
 		| paste -s -d, -
         )
         sg_id=$(
-            aws ec2 describe-security-groups \
-                --filters "Name=vpc-id,Values=${vpc_id}" \
-                          "Name=group-name,Values=${environment}-elasticsearch" \
-                | egrep "^\|\|  GroupId" \
-                | tr -d ' ' \
-                | cut -d\| -f4
+		capcom list \
```
Comment (ifosch) [inline bin/lib]: Maybe the README should have capcom as a requirement ;P

**Comment by ifosch** [comment]
+1 but consider my comment about the README, please.

File: bin/lib (hunk 1 above)
Comment (dsager) [inline bin/lib]: capcom made street fighter, right?

**Comment by dsager** [comment]
+1

**Comment by christofdamian** [comment]
+1

**Comment by bbrassart** [comment]
+1

**Comment by yaneq** [comment]
+1

**Pull request merged**